src/common/collection.c

```diff
@@ -1211,19 +1211,67 @@ gboolean dt_collection_hint_message_internal(void *message)
 
 void dt_collection_hint_message(const dt_collection_t *collection)
 {
+  /* if relevant, determine offset of selection */
+  GList *selected_imgids = dt_collection_get_selected(collection, 1);
+  int selected_imgids_count = g_list_length(selected_imgids);
+  int selected = -1;
+
+  if(selected_imgids_count > 0)
```

Comment:
I personally would use `if(selected_imgids)` and not use `g_list_length()` at all. But that is just me trying to limit function calls. 😄